src/DataDepsGenerators.jl

```diff
@@ -87,6 +87,48 @@ include("DataCite.jl")
 include("Figshare.jl")
 include("JSONLD/JSONLD.jl")
 
+function aggregate(generator_meta)
```

Comment:
`generator_meta` is not a good variable name.
the importance of `generator` is not clear.
and it is not a plural.
variables that contain lists, should be plural; or a least names such that they are obviously not just a single value.


Maybe you want `metadatas`